src/ONYXKEYS.ts

```diff
@@ -469,7 +469,7 @@ type OnyxValues = {
     [ONYXKEYS.COLLECTION.REPORT]: OnyxTypes.Report;
     [ONYXKEYS.COLLECTION.REPORT_METADATA]: OnyxTypes.ReportMetadata;
     [ONYXKEYS.COLLECTION.REPORT_ACTIONS]: OnyxTypes.ReportActions;
-    [ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS]: OnyxTypes.ReportActionsDrafts;
+    [ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS]: Record<string, OnyxTypes.ReportActionsDrafts>;
```

Comment:
```suggestion
    [ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS]: OnyxTypes.ReportActionsDrafts;
```

This is the structure of `REPORT_ACTIONS_DRAFTS`:

![Screenshot 2024-02-09 at 08 04 21](https://github.com/Expensify/App/assets/20051562/00ddab33-3ee0-4b09-8d9f-9e921cfb7d0a)

Given that `ReportActionsDrafts` is already a record of drafts (`Record<string, ReportActionsDraft>`), this type should be only `OnyxTypes.ReportActionsDrafts`

Reply:
I logged what I am getting from `useReportActionsDrafts` and it looks like this 
<img width="376" alt="image" src="https://github.com/Expensify/App/assets/25230417/b3dd23b5-864d-4c9e-a9ab-a3d69d3451bb">
so for me it seems to be `Record<string, ReportActionsDraft>`, so I think what I passed is good?

Reply:
It's probably the way `useReportActionsDrafts` is returning, because in the storage is the way I showed in the screenshoot

Reply:
Could you check your local storage?

Reply:
yep in LS it looks like you said so I belive I need to adjust the type of the hooks from `createOnyxContext`